.gitignore

```diff
@@ -31,4 +31,7 @@ package-lock.json*
 /playwright/.cache/
 
 #firebase
-.firebaserc
\ No newline at end of file
+.firebaserc
+*.log
```

Comment:
why we have logs in the root? is the seeding script adding them? if so, I would config the seeding script to write them in a `log` directory

Reply:
We have log generated from Firebase emulator, it isn't related directly to seeding.